app-rails/app/adapters/auth/cognito_adapter.rb

```diff
@@ -14,7 +14,7 @@ def initialize(client: Aws::CognitoIdentityProvider::Client.new)
   def create_account(email, password)
     begin
       response = @client.sign_up(
-        client_id: ENV["AWS_COGNITO_CLIENT_ID"],
+        client_id: ENV["COGNITO_CLIENT_ID"],
```

Comment:
not sure if this is the right rails way, but could we set this  client_id, along with the user_pool_id, from the ENV at the top of the class and reuse the variable, instead of resetting it in each method to be more DRY?

Reply:
I think this is a good change, but I'd like to break it out into a separate issue.